src/EditorFeatures/Core/Preview/DifferenceViewerPreview.cs

```diff
@@ -33,6 +48,9 @@ public void Dispose()
             }
 
             _viewer = null;
+
+            LeftWorkspace?.Dispose();
+            RightWorkspace?.Dispose();
```

Comment:
no need for `?.` afaict.

Reply:
➡️ These can be null. See 6a08a737958648f643d2aed618603b02c895e1c3

Reply:
➡️ Covered by 6a08a737958648f643d2aed618603b02c895e1c3